packages/salesforcedx-vscode-core/test/commands/forceOrgCreate.test.ts

```diff
@@ -18,45 +18,99 @@ import { nls } from '../../src/messages';
 // tslint:disable:no-unused-expression
 describe('Force Org Create', () => {
   describe('Alias Gatherer', () => {
+    const EVENT_CANCEL = 'CANCEL';
+    const EVENT_CONTINUE = 'CONTINUE';
     const TEST_ALIAS = 'testAlias';
     const TEST_WORKSPACE = 'sfdxsimple'; // FYI: This test uses the workspace created by the system tests to run
+    const TEST_ORG_EXPIRATION_DAYS = '7';
+    const TEST_ORG_EXPIRATION_DAYS_PLUS_ONE_DAY = '8';
+    const TEST_ORG_EXPIRATION_DAYS_INPUT_FLOAT = '8.2';
+    const TEST_ORG_EXPIRATION_DAYS_INPUT_INVALID_RANGE = '31';
     let inputBoxSpy: sinon.SinonStub;
 
-    before(() => {
+    beforeEach(() => {
       inputBoxSpy = sinon.stub(vscode.window, 'showInputBox');
-      inputBoxSpy.onCall(0).returns(undefined);
-      inputBoxSpy.onCall(1).returns('');
-      inputBoxSpy.onCall(2).returns(TEST_ALIAS);
     });
 
-    after(() => {
+    afterEach(() => {
       inputBoxSpy.restore();
     });
 
     it('Should return cancel if alias is undefined', async () => {
+      inputBoxSpy.onCall(0).returns(undefined);
       const gatherer = new AliasGatherer();
       const response = await gatherer.gather();
-      expect(inputBoxSpy.calledOnce).to.be.true;
-      expect(response.type).to.equal('CANCEL');
+      expect(inputBoxSpy.callCount).to.equal(1);
```

Comment:
Is there a reason to change it from `expect(inputBoxSpy.calledOnce).to.be.true` to `expect(inputBoxSpy.callCount).to.equal(1)`. I just want to understand if this is a preference or is there is a technical reason.

Reply:
@vazexqi You'd have to look at the intermediate changes. There was previously a running total of count that went through all of the tests that was part of the original PR. I didn't actually change the .to.be.true to a count, myself. When I refactored before/after to beforeEach and afterEach everything was already checking the count for the number of calls (which, in most cases was a total of 2) and so I just went uniform checking the call count the same way through everything instead of calledOnce or calledTwice etc.